rclpy/rclpy/node.py

```diff
@@ -761,6 +763,33 @@ def _set_parameters_atomically(
 
         return result
 
+    def add_on_set_parameters_callback(
+        self,
+        callback: Callable[[List[Parameter]], SetParametersResult]
+    ) -> None:
+        """
+        Add a callback in front to the list of callbacks.
+
+        Calling this function will add a callback in self._parameter_callbacks list.
+
+        :param callback: The function that is called whenever parameters are set for the node.
+        """
+        self._parameters_callbacks.insert(0, callback)
+
+    def remove_on_set_parameters_callback(
+        self,
+        callback: Callable[[List[Parameter]], SetParametersResult]
+    ) -> None:
+        """
+        Remove a callback from list of callbacks.
+
+        Calling this function will remove the callback from self._parameter_callbacks list.
+
+        :param callback: The function that is called whenever parameters are set for the node.
+        :raises: ValueError if a callback is not present in the list of callback.
```

Comment:
Nitpick (as mentioned before): ... list of callbackS.

Reply:
Done sir.